components/admin/admin-accessibility.tsx

```
@@ -0,0 +1,231 @@
+
+// Accessible status announcer for screen readers
+export function AdminStatusAnnouncer({ 
+  message, 
+  priority = 'polite' 
+}: { 
+  message: string; 
+  priority?: 'polite' | 'assertive';
+}) {
+  return (
+    <div
+      role="status"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)